roles/edpm_nova/tasks/main.yml

```diff
@@ -14,6 +14,18 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Verify dependencies
+  block:
+    - name: Check for ssh_known_hosts file
+      ansible.builtin.stat:
+        path: /etc/ssh/ssh_known_hosts
+      register: _ssh_known_hosts
+
+    - name: Fail if ssh_known_hosts file is missing
+      ansible.builtin.fail:
+        msg: "Nova depends on ssh-known-hosts OpenStackDataPlaneService"
+      when: not _ssh_known_hosts.stat.exists | bool
+
```

Comment:
couldn't we just touch the file (as root) if it does't exist? I guess there might be some valid reasons to omit using the ssh-known-hosts service.